public/app/features/dashboard/utils/getPanelMenu.ts

```diff
@@ -202,8 +206,50 @@ export function getPanelMenu(
     subMenu: inspectMenu,
   });
 
+  const createAlert = async () => {
+    const formValues = await panelToRuleFormValues(panel, dashboard);
+
+    const ruleFormUrl = urlUtil.renderUrl('/alerting/new', {
+      defaults: JSON.stringify(formValues),
+      returnTo: location.pathname + location.search,
+    });
+
+    locationService.push(ruleFormUrl);
+  };
+
+  const navigateToAlertListView = async () => {
+    const alertListUrl = urlUtil.renderUrl('/alerting/list', {
+      returnTo: location.pathname + location.search,
+    });
+
+    locationService.push(alertListUrl);
+  };
+
+  const onNavigateToAlertListView = (event: React.MouseEvent) => {
```

Comment:
This one makes no sense from the panel menu

Reply:
I've just remove it :-)